platforms/hyperledger-indy/charts/indy-auth-job/values.yaml

```diff
@@ -0,0 +1,30 @@
+metadata:
+  namespace:
+  #baf
+  name:
+  #indy-auth-job
+
+network:
+  name:
+
+image:
+  name:
+  #indy-auth-job
+  repository:
+  #alpine:3.9.4
+
+vault:
```

Comment:
I think we'll need the vault policy, role and serviceaccount of the vault auth we're going to create as input job params as well

Reply:
added